ci-operator/step-registry/ipi/conf/aws/oidc-creds-provision/ipi-conf-aws-oidc-creds-provision-ref.yaml

```diff
@@ -1,6 +1,7 @@
 ref:
   as: ipi-conf-aws-oidc-creds-provision
-  from: cli
+  from: cloud-credential-operator
+  cli: latest
```

Comment:
This will have changes from the PR right? We want to test any changes to ccoctl as well. I am not sure if the current code does that.

Reply:
you will still have new versions of both `oc` and `ccoctl` in the step, both as `master` stands before this PR, and as I have things set up in the PR.  The PR just gets us there more conveniently, with less in-step hoop-jumping.